infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/server/OpenShiftServerExposer.java

```diff
@@ -91,95 +87,22 @@
  */
 public class OpenShiftServerExposer extends KubernetesServerExposer<OpenShiftEnvironment> {
 
-  private final OpenShiftEnvironment openShiftEnvironment;
-
   public OpenShiftServerExposer(
-      String machineName, Pod pod, Container container, OpenShiftEnvironment openShiftEnvironment) {
-    super(Collections.emptyMap(), machineName, pod, container, openShiftEnvironment);
-    this.openShiftEnvironment = openShiftEnvironment;
+      ExternalServerExposerStrategy OpenshiftExternalServerExposerStrategy,
```

Comment:
Please generify this variable with OpenShiftEnvironment. 
Also please rename this variable, it's quite unusual for java to name variable with first upper character.

Reply:
done